recipes/quantms-utils/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "quantms-utils" %}
+{% set version = "0.0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/quantms_utils-{{ version }}.tar.gz
+  sha256: e6143d77a85b5a9d716f8f5fd7286b012827591e9db5d62471f76456d4b2caf4
+
+build:
+  entry_points:
+    - quantmsutilsc=quantmsutils.quantmsutilsc:main
+  noarch: python
+  run_exports:
+    - {{ pin_subpackage(name, max_pin="x.x") }}
```

Comment:
```suggestion
    - {{ pin_subpackage('quantms-utils', max_pin="x.x") }}
```

Reply:
Why ?
Using 'name' variable is preferable

Reply:
Do we have this as a recommendation? I think this is not very readable ... but then this entire line is hard to understand :)

Reply:
> I think this is not very readable

The same could be said for `source > url` but everyone practices it, e.g. `    - 'https://depot.galaxyproject.org/software/bioconductor-{{ name }}/bioconductor-{{ name }}_{{ version }}_src_all.tar.gz'` :-)
I guess it is just a personal preference...